lib/validate-colors.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+/**
+ * Module dependencies
+ */
+
+var isValidRule = require('./is-valid-rule');
+
+/**
+ * Module exports
+ */
+
+module.exports = validateColors;
+
+/**
+ * @param {Array} rules
+ */
+
+function validateColors(rules) {
+  rules.forEach(function (rule) {
+    if (!isValidRule(rule)) return;
+
+    rule.declarations.forEach(function (declaration) {
+      if (declaration.type !== 'declaration') {
+        return;
+      }
+
+      if (declaration.property.indexOf("--") !== -1) {
+        return;
+      }
+
+      var column = declaration.position.start.column;
+      var line = declaration.position.start.line;
+      var value = declaration.value;
+
+      var containsVariable = declaration.value.indexOf("--") === -1 || declaration.value.indexOf("var(") === -1;
```

Comment:
same comment as previous re: `var(--`. also you cached `value` but still use `declaration.value`.